preferences.php

```diff
@@ -49,9 +49,12 @@
 
 echo $OUTPUT->header();
 
+// Save the preferences.
```

Comment:
This comment is good, I remember looking for "where is saving the preferences" before so that actually helps where to look.